.github/workflows/release-deploy-all.yml

```diff
@@ -68,7 +64,7 @@ on:
 
 env:
   NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
-  BUILD_REF: ${{ inputs.gitRef }}
+  BUILD_REF: ${{ github.ref_name }}
```

Comment:
For information, this one would be a problem if we use the `github.ref` instead of `github.ref_name` because it is used internally to name resources deployed to Openshift, I am not wrong.